File: src/main/java/com/tobe/healthy/lessonhistory/repository/LessonHistoryRepository.java
package com.tobe.healthy.lessonhistory.repository;

import org.springframework.data.jpa.repository.JpaRepository;

import com.tobe.healthy.lessonhistory.domain.entity.LessonHistory;

public interface LessonHistoryRepository extends JpaRepository<LessonHistory, Long>, LessonHistoryRepositoryCustom {
	LessonHistory findByIdAndTrainerId(Long lessonHistoryId, Long trainerId);
}

File: src/main/kotlin/com/tobe/healthy/lessonHistory/repository/LessonHistoryRepository.kt
package com.tobe.healthy.lessonHistory.repository

import com.tobe.healthy.lessonHistory.domain.entity.LessonHistory
import org.springframework.data.jpa.repository.JpaRepository

interface LessonHistoryRepository : JpaRepository<LessonHistory, Long>, LessonHistoryRepositoryCustom {
}
